sources/install/package_base.sh

```diff
@@ -244,6 +244,7 @@ function install_gf() {
 function post_install() {
     # Function used to clean up post-install files
     colorecho "Cleaning..."
+    apt update
```

Comment:
Why running an `apt update` here?
And wouldn't `apt-get update` be more suited for a script usage

Reply:
Because when you launch a new container, apt is not up to date and you cannot install a packet without doing apt update. I haven't really thought about which one is better between apt update and apt-get update

Reply:
According to the [manual page](https://manpages.debian.org/buster/apt/apt.8.en.html), in scripts apt-get must be preferred, while apt is for interactive sessions.

Reply:
That's right !

Reply:
100% agree with @gbe 
Let’s remove the apt update here